linetools/isgm/utils.py

```diff
@@ -270,7 +271,7 @@ def complist_from_table(table):
 
     Notes
     -----
-    Mandatory column names: 'RA', 'DEC', 'ion_name', 'z_comp', 'vmin', 'vmax'
+    Mandatory column names: 'RA', 'DEC', 'ion_name', 'z', 'vmin', 'vmax'
```

Comment:
should be `z_comp`, to differenciate from `Z`.

Reply:
yup